apps/core/src/router/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Layout } from "@zuri/ui";
+import { createBrowserRouter } from "react-router-dom";
+import { ErrorPage } from "../pages";
```

Comment:
I see where the mention of re-exporting came from
I apologize for that

can you remove this one(and also the pages/index.ts that re-exports it)?

Reply:
noted